FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -72,12 +79,15 @@ class IssuesViewController : UITableViewController {
     }
     
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        return .lightContent
+        return shouldFetchAllIssues ? .default : .lightContent
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         logs = ContactLogs.load()
+        if shouldFetchAllIssues {
+            navigationController?.setNavigationBarHidden(false, animated: animated)
+        }
```

Comment:
I'm tempted to suggest a subclass for this.  The `AllIssuesViewController` could have all of these methods defined as simple overrides, including the fetch call.

@mccarron thoughts on this?